bip-0388.mediawiki

```diff
@@ -1,3 +1,6 @@
+RECENT CHANGES:
+* (06 Nov 2024) Added support for <tt>musig</tt> key placeholders in descriptor templates
+
```

Comment:
The BIP formatting rules prescribe that the document starts with the preamble. May I suggest that you introduce a Change Log section in the [style of BIP 327](https://github.com/bitcoin/bips/blob/master/bip-0327.mediawiki#change-log) or [BIP 352](https://github.com/bitcoin/bips/blob/master/bip-0352.mediawiki#change-log) toward the end of the document where you incorporate this note?

Reply:
@murchandamus yes, see https://github.com/bitcoin/bips/pull/1697#discussion_r1834640167

@bigspider fwiw a suggested order at https://github.com/bitcoin/bips/pull/1691#discussion_r1817926795

Reply:
Good suggestion, done in eec10546357af967978a13aa0c989a31d4309b17.